argoverse/evaluation/detection/utils.py

```diff
@@ -323,7 +319,7 @@ def filter_instances(
     return filtered_instances
 
 
-def rank(dts: List[ObjectLabelRecord]) -> Tuple[np.ndarray, np.ndarray]:
+def rank(dts: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
Hi @benjaminrwilson , since the function signature is a bit less informative now (albeit more correct), could you expand the `dts` description to:
```python
    Args:
        dts: Object array of shape (N,) representing detections. Each object is an `ObjectLabelRecord`
```